src/test/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerGenerateSupplierXmlServiceImplTest.java

```diff
@@ -126,6 +126,46 @@ void testDateFormatters() {
         assertEquals("2023-05-17T17:48:45.727Z", actualHeaderDateString);
     }
     
+    @Test
+    void testDateFormatterLocaleFileName() {
+        Clock myClock = Clock.systemDefaultZone();
+        LocalDateTime dayAndTimeNow = LocalDateTime.now(myClock);
+        
+        String manualFileNameDateString = new String();
+        int myNanoSeconds = dayAndTimeNow.getNano() / 1000000;
+        manualFileNameDateString = manualFileNameDateString + dayAndTimeNow.getYear() 
```

Comment:
Perhaps these two new tests could be simplified by using a String formatter and a constant date for testing?